### PR TITLE
Examples: Fix controls in webgl_multiple_scenes_comparison.

### PR DESCRIPTION
**Pull request by Mugen87**
see https://github.com/mrdoob/three.js/pull/20194#issuecomment-683403233

### CODE REVIEW

File: examples/webgl_multiple_scenes_comparison.html [hunk 1]
```diff
@@ -7,6 +7,10 @@
 		<link type="text/css" rel="stylesheet" href="main.css">
 		<style>
 
+			html, body {
+				touch-action: none;
```
Comment (Mugen87) [inline examples/webgl_multiple_scenes_comparison.html:11]: I was not able to fix touch controls without this CSS, see https://stackoverflow.com/questions/48124372/pointermove-event-not-working-with-touch-why-not

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev35/examples/webgl_multiple_scenes_comparison.html

File: examples/webgl_multiple_scenes_comparison.html (hunk 1 above)
Comment (mrdoob) [reply examples/webgl_multiple_scenes_comparison.html:11]: I also battled with this today. Safari/iOS doesn't support this.

I ended up doing this instead:

```js
body.addEventListener( 'touchstart', function ( event ) {

    event.preventDefault(); // prevent scrolling

} );
```

Comment (Mugen87) [reply examples/webgl_multiple_scenes_comparison.html:11]: If I add this code, I get the following runtime error in Chrome:

> webgl_multiple_scenes_comparison:190 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080

It only seems to work if I mark the listener as non-passive:
```js
window.addEventListener( 'touchstart', function ( event ) {

    event.preventDefault(); // prevent scrolling

}, { passive: false } );
```
Not sure if this is appropriate...

Comment (mrdoob) [reply examples/webgl_multiple_scenes_comparison.html:11]: Ah, maybe `document.body` instead?

This `touchstart` and `touch-action` gotcha is such a pain...

Maybe in this case `touch-action` is better actually.

Comment (Mugen87) [reply examples/webgl_multiple_scenes_comparison.html:11]: > Ah, maybe document.body instead?

Unfortunately, I see the same error with `document.body`.

Comment (Mugen87) [reply examples/webgl_multiple_scenes_comparison.html:11]: >  Safari/iOS doesn't support this.

Doesn't this limitation prevent a complete migration of the controls like `OrbitControls` to `PointerEvents`? According to the stackoverflow thread it seems that `pointermove` on touch devices will only work if you set `touch-action` to `none`.

Comment (mrdoob) [reply examples/webgl_multiple_scenes_comparison.html:11]: In `TransformControls` I did this:

```js
domElement.addEventListener( 'touchstart', function ( event ) {

    event.preventDefault(); // prevent scrolling

} );
```

Comment (Mugen87) [reply examples/webgl_multiple_scenes_comparison.html:11]: Um, why does this approach work with `TransformControls` but not in the example 🤔 ?

Comment (mrdoob) [reply examples/webgl_multiple_scenes_comparison.html:11]: Maybe because I'm adding it to the `renderer.domElement` directly?

Comment (Mugen87) [reply examples/webgl_multiple_scenes_comparison.html:11]: I've tested this right now. The errors go away but the UI behavior is still the same. Meaning the `pointermove` gets canceled after a short usage.

It only works if I add all event listeners to `renderer.domElement`. But then it's not possible to just drag the `slider` element but the entire overlay.

Comment (mrdoob) [reply examples/webgl_multiple_scenes_comparison.html:11]: Okay, I'll merge this and play around with the code 👍

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
I ended up doing quite a bit of clean up: https://github.com/mrdoob/three.js/commit/65494f3b0973b39d851dc18859745b708451df65

But the solution was adding the listener to `slider`:

```js
slider.addEventListener( 'touchstart', function ( event ) {

	event.preventDefault(); // prevent scrolling

} );
```

I have not tested if `slider.style.touchAction = 'none'` works yet... I'll test that later today.

**Comment by mrdoob** [comment]
Okay, seems like `slider.style.touchAction = 'none'` works. At least in the latest Safari/iOS...

I'll update the code.

**Comment by mrdoob** [comment]
https://github.com/mrdoob/three.js/commit/fef0b7a82c0b70af0090039ca66e8bfda46c11d8 https://github.com/mrdoob/three.js/commit/a72e73d74e2f71f3ac05e6a73870acfd5f28ffc5

**Pull request merged**